Make UMAP plotting optional (#159)
AlexsLemonade/scpcaTools: inst/rmd/qc_report.rmd

}
```

> Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
> The raw counts from all remaining cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 

Reviewer: I started with editing these sentences to refer to the fact that miQC is not always done, but I went back to looking at it as a whole, and I think we can probably just drop these two sentences? Back on line 426, we say "The below plot highlights cells that were removed prior to normalization and dimensionality reduction." So maybe that covers it?

```suggestion
The raw counts from all cells that remain after filtering low quality cells are then normalized prior to selection of highly variable genes and dimensionality reduction. 
```
